core/src/main/scala/chisel3/experimental/dataview/DataProduct.scala

```diff
@@ -82,7 +101,7 @@ object DataProduct extends LowPriorityDataProduct {
   }
 
   /** [[DataProduct]] implementation for any `Seq[A]` where `A` has an implementation of `DataProduct`. */
-  implicit def seqDataProduct[A: DataProduct]: DataProduct[Seq[A]] = new DataProduct[Seq[A]] {
+  def seqDataProduct[A: DataProduct]: DataProduct[Seq[A]] = new DataProduct[Seq[A]] {
```

Comment:
It seems like this is no longer implicit because the `IterableOnce` version supersedes it. Should this be deprecated and slated for removal?

Reply:
Probably, I'll add a deprecation